pkg/upstream/upgrade.go

```diff
@@ -70,33 +66,6 @@ func Upgrade(appSlug string, options UpgradeOptions) (*UpgradeResponse, error) {
 		}
 		defer os.RemoveAll(airgapRootDir)
 
-		registryEndpoint := options.RegistryEndpoint
-		registryNamespace := options.RegistryNamespace
-		registryUsername := options.RegistryUsername
-		registryPassword := options.RegistryPassword
-
-		if registryNamespace == "" {
```

Comment:
this (and the code below) already happens as part of the `getRegistryConfig` function